src/service/alerts/alert.rs

```diff
@@ -415,6 +415,10 @@ pub async fn create<C: TransactionTrait>(
     prepare_alert(org_id, &stream_name, &alert_name, &mut alert, true).await?;
 
     let alert = db::alerts::alert::create(conn, org_id, folder_id, alert).await?;
+    if let Some(id) = alert.id.map(|id| id.to_string()) {
+        set_ownership(&alert.org_id, "alerts", Authz::new(&id)).await;
+    }
```

Comment:
Once the application switches over to using the V2 alert routes, alerts are only created with the `id` identifier in OpenFGA. Any alerts created with the V2 routes won't need to be migrated.